### DISCUSSION TITLE
Senior Technical Writer (Documentation) position

### DISCUSSION THREAD

**Discussion by leroberto8** [prisma/prisma#9815] [Q&A] [answered]
how the daily job look like, please ? I am curious I would like to start working.

**Comment by tanberry** [comment, marked as answer]
Hi @leroberto8 , the [job description and requirements](https://boards.greenhouse.io/prisma/jobs/5608988002) are posted, please give them a review.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!

**Comment by leroberto8** [comment]
Hello !
I would like to take part of the  of recruitment process again : Re:
[prisma/prisma] Senior Technical Writer (Documentation) positionplease let
me know if it is possible.

 (Discussion #9815)

Bien cordialement/ Best regards/ Pozdrawiam serdecznie,
Robert Mbogni

On Wed, 22 Jan 2025 at 08:13, Raphael Etim ***@***.***> wrote:
Closed #9815 as resolved.—
